collaborations/dark-tides-of-bilgewater/barbarian-depths.xml

```diff
@@ -0,0 +1,134 @@
+		</description>
+		<sheet>
+			<description>You have the ability to breathe underwater.</description>
+		</sheet>
+		<rules>
+			<stat name="speed:swim" value="speed" bonus="base"/>
+		</rules>
+	</element>
+
+	<element name="Dredge Line" type="Archetype Feature" source="Legends of Runeterra: Dark Tides of Bilgewater" id="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_PATH_OF_THE_DEPTHS_DREDGE_LINE">
+		<description>
+			<p>Starting when you choose this path at 3rd level, you manifest an extra appendage when you enter your rage. This weapon can appear as a kraken tentacle, a giant anchor, preternatural jaws, or something else based on your history.</p>
+			<p class="indent">As a bonus action, you can use this appendage to strike at one creature of your choice that you can see within 15 feet. The target must succeed on a Strength saving throw (DC equal to 8 + your proficiency bonus + your Strength modifier) or be pulled up to 10 feet in a straight line towards you.</p>
+		</description>
+		<sheet>
+			<description>You manifest an extra appendage when you enter your rage. As a bonus action, you can use this appendage to strike at one creature of your choice that you can see within 15 feet. The target must succeed on a Strength saving throw (DC equal to 8 + your proficiency bonus + your Strength modifier) or be pulled up to 10 feet in a straight line towards you.</description>
+		</sheet>
+	</element>
+
+	<element name="Ghostwater Dive" type="Archetype Feature" source="Legends of Runeterra: Dark Tides of Bilgewater" id="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_PATH_OF_THE_DEPTHS_GHOSTWATER_DIVE">
+		<description>
+			<p>Beginning at 6th level, you can burst into water then materialize somewhere else as an action. You magically teleport along with any equipment you are wearing or carrying, up to 30 feet to an unoccupied space you can see. Before or after teleporting, you can make one attack, as part of your action. Moving in this way does not provoke opportunity attacks.</p>
+		</description>
+		<sheet>
+			<description>You can burst into water then materialize somewhere else as an action. You magically teleport along with any equipment you are wearing or carrying, up to 30 feet to an unoccupied space you can see. Before or after teleporting, you can make one attack, as part of your action. Moving in this way does not provoke opportunity attacks.</description>
+		</sheet>
+	</element>
+
+	<element name="Manifestations of the Deep" type="Archetype Feature" source="Legends of Runeterra: Dark Tides of Bilgewater" id="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_PATH_OF_THE_DEPTHS_MANIFESTATIONS_OF_THE_DEEP">
+		<description>
+			<p>At 10th level, you can manifest additional adaptations of the deep. Select one of the below adaptations you manifest, during a long rest you may replace your chosen manifestation with a new option from this list.</p>
+		</description>

```

Comment:
Missing sheet description, which can just not have "At 10th level, etc." Can be reworded better especially when talking about a "list"